applications/main/subghz/subghz_cli.c

```diff
@@ -196,6 +196,50 @@ void subghz_cli_command_tx(Cli* cli, FuriString* args, void* context) {
     subghz_environment_free(environment);
 }
 
+#include "fild_validation_rfid_driver.h"
+#include <furi_hal_nfc.h>
+#include <lib/ST25RFAL002/include/rfal_nfc.h>
+#include <lib/ST25RFAL002/source/st25r3916/st25r3916_com.h>
+void subghz_cli_command_us(Cli* cli, FuriString* args, void* context) {
+    UNUSED(context);
+    UNUSED(args);
+
+    printf("Check frecuency. Press CTRL+C to stop\r\n");
```

Comment:
```suggestion
    printf("Check frequency. Press CTRL+C to stop\r\n");
```
and other occurrences

Reply:
this is a test build, it won't be in the main, but thanks anyway